.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+Please provide a brief description of the changes made in this pull request.
+
+## Checklist
+
+Please make sure that the following checkpoints have been completed before submitting this pull request:
+
+- [ ] Have you followed the code convention guidelines specified in the repository?
+- [ ] Have you included the updated method/function documentation in the `documentation.md` file?
+
```

Comment:
Add one more checkpoint here, that is
+ - [ ] Have you included the method in the `endpoints. md` file.

Reply:
Also add a section like
+ ### Issue number.